src/main/webapp/dashboard.js

```diff
@@ -1,34 +1,37 @@
+import {getUser} from 'user.js';
```

Comment:
The linter says `[no-unused-vars] 'getUser' is defined but never used.` It looks like this is used in the commented-out code below. Either uncomment the commented-out code or delete the commented-out code and this unused import.